compiler/arm/codegen/Instruction.hpp

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
```

Comment:
This file should be the Instruction header that connects the `TR` namespace with the `OMR` namespace only.  It should not be providing any implementation as it currently does, just simply pass through the `TR` namespace implemetation to the `OMRConnector` implementation.